src/extensions/php-fpm-metrics/pkg/phpfpm/status_test.go

```diff
@@ -0,0 +1,92 @@
+package phpfpm
```

Comment:
in general I highly recommend keeping tests in a separate `*_tests` package.

From [The Zen of Go](https://the-zen-of-go.netlify.app/):

> *Write tests to lock in the behaviour of your package’s API*
>
> Test first or test later, if you shoot for 100% test coverage or are happy with less, regardless your package’s API is your contract with its users. Tests are the guarantees that those contracts are written in. Make sure you test for the behaviour that users can observe and rely on.

When our tests are in the same package as the code, we're able to use things outside the packages API, and potentially miss things.

Reply:
This usually requires design changes to add dependency injection or the functional options pattern to support mocking.